### PR TITLE
Run tests with Travis.

### PR DESCRIPTION
**Pull request by kaspth**
Just to make sure I didn't miss anything :grin:

cc @rafaelfranca @dhh

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  irc:
+    on_success: change
+    on_failure: always
+    channels:
+      - "irc.freenode.org#rails-contrib"
+  campfire:
+    on_success: change
+    on_failure: always
+    rooms:
+      - secure: "HnqK+r2/SLv2Z1XMkL0oNO0w/xaUn9cGHpr2mGDNnk+SUz1XBO2bu5c0ykyAVnLienFPCDExX3RYwTMNvC4AhkAR8DK0v4FD+0g/dMy2j1wGzQlNtqF2B4EbRpVpZtmM3hK6dDNRR3V/zipqTkut00PoFPp/Bzm7DoH/w/MIPBY="
```
Comment (rafaelfranca) [inline .travis.yml]: This needs to change. Let me grab the right token

Comment (rafaelfranca) [inline .travis.yml]:
```
"EZmqsgjEQbWouCx6xL/30jslug7xcq+Dl09twDGjBs369GB5LiUm17/I7d6H1YQFY0Vu2LpiQ/zs+6ihlBjslRV/2RYM3AgAA9OOC3pn7uENFVTXaECi/io1wjvlbMNrf1YJSc3aUyiWKykRsdZnZSFszkDs4DMnZG1s/Oxf1JTYEGNWW3WcOFfYkzcS7NWlOW9OBf4RuzjtLYF05IO4t4FZI1aTWrNV3NNMZ+tqmiQHHNrQE/CzQE3ujqFiea2vVZ7PwvmjVWJgC29UZqS7HcNuq6cCMtMZZuubCZmyT85GjJ/SKTShxFqfV1oCpY3y6kyWcTAQsUoLtPEX0OxLeX+CgWNIJK0rY5+5/v5pZP1uwRsMfLerfp2a9g4fAnlcAKaZjalOc39rOkJl8FdvLQtqFIGWxpjWdJbMrCt3SrnnOccpDqDWpAL798LVBONcOuor71rEeNj1dZ6fCoHTKhLVy6UVm9eUI8zt1APM0xzHgTBI1KBVZi0ikqPcaW604rrNUSk8g/AFQk0pIKyDzV9qYMJD2wnr42cyPKg0gfk1tc9KRCNeH+My1HdZS6Zogpjkc3plAzJQ1DAPY0EBWUlEKghpkyCunjpxN3cw390iKgZUN52phtmGMRkyNnwI8+ELnT4I+Jata1mFyWiETM85q8Rqx+FeA0W/BBsEAp8="
```

**Comment by rafaelfranca** [comment]
It is running. :shipit:

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  irc:
+    on_success: change
+    on_failure: always
+    channels:
+      - "irc.freenode.org#rails-contrib"
+  campfire:
+    on_success: change
+    on_failure: always
+    rooms:
+      - secure: "EZmqsgjEQbWouCx6xL/30jslug7xcq+Dl09twDGjBs369GB5LiUm17/I7d6H1YQFY0Vu2LpiQ/zs+6ihlBjslRV/2RYM3AgAA9OOC3pn7uENFVTXaECi/io1wjvlbMNrf1YJSc3aUyiWKykRsdZnZSFszkDs4DMnZG1s/Oxf1JTYEGNWW3WcOFfYkzcS7NWlOW9OBf4RuzjtLYF05IO4t4FZI1aTWrNV3NNMZ+tqmiQHHNrQE/CzQE3ujqFiea2vVZ7PwvmjVWJgC29UZqS7HcNuq6cCMtMZZuubCZmyT85GjJ/SKTShxFqfV1oCpY3y6kyWcTAQsUoLtPEX0OxLeX+CgWNIJK0rY5+5/v5pZP1uwRsMfLerfp2a9g4fAnlcAKaZjalOc39rOkJl8FdvLQtqFIGWxpjWdJbMrCt3SrnnOccpDqDWpAL798LVBONcOuor71rEeNj1dZ6fCoHTKhLVy6UVm9eUI8zt1APM0xzHgTBI1KBVZi0ikqPcaW604rrNUSk8g/AFQk0pIKyDzV9qYMJD2wnr42cyPKg0gfk1tc9KRCNeH+My1HdZS6Zogpjkc3plAzJQ1DAPY0EBWUlEKghpkyCunjpxN3cw390iKgZUN52phtmGMRkyNnwI8+ELnT4I+Jata1mFyWiETM85q8Rqx+FeA0W/BBsEAp8="
```
Comment (kaspth) [inline .travis.yml]: @rafaelfranca long token, much secure, wow :+1:

**Comment by rafaelfranca** [comment]
Seems like the Gemfile.lock is broken again. Or we remember to always update it or lets remove from the repository.

File: .travis.yml (hunk 2 above)
Comment (rafaelfranca) [inline .travis.yml]: Each repository has its own token so we can't copy from other repository, this is why I generated one just for this repository.

Comment (rafaelfranca) [inline .travis.yml]:
```
travis encrypt -r rails/actioncable TOKEN_INFORMATION
```

Comment (kaspth) [inline .travis.yml]: Ah, I see. I was thinking that wouldn't be very secure otherwise :smile:

**Pull request merged**